cilium monitor crash

```
panic: Unable to get object: no such file or directory

goroutine 1 [running]:
panic(0x13ac9e0, 0xc421411bc0)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/cilium/cilium/cilium/cmd.runMonitor()
	/home/tgraf/dev/go/src/github.com/cilium/cilium/cilium/cmd/monitor.go:99 +0x3e4
github.com/cilium/cilium/cilium/cmd.glob..func14(0x1ff10a0, 0x2599aa0, 0x0, 0x0)
	/home/tgraf/dev/go/src/github.com/cilium/cilium/cilium/cmd/monitor.go:42 +0x14
github.com/cilium/cilium/vendor/github.com/spf13/cobra.(*Command).execute(0x1ff10a0, 0x2599aa0, 0x0, 0x0, 0x1ff10a0, 0x2599aa0)
	/home/tgraf/dev/go/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:648 +0x443
github.com/cilium/cilium/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x1fee840, 0xc420062058, 0x0, 0x467dee)
	/home/tgraf/dev/go/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:735 +0x367
github.com/cilium/cilium/vendor/github.com/spf13/cobra.(*Command).Execute(0x1fee840, 0x0, 0x0)
	/home/tgraf/dev/go/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:693 +0x2b
github.com/cilium/cilium/cilium/cmd.Execute()
	/home/tgraf/dev/go/src/github.com/cilium/cilium/cilium/cmd/root.go:69 +0x31
main.main()
	/home/tgraf/dev/go/src/github.com/cilium/cilium/cilium/main.go:20 +0x14
```

--- a/cilium/cmd/monitor.go
+++ b/cilium/cmd/monitor.go
@@ -94,6 +94,11 @@ func receiveEvent(msg *bpf.PerfEventSample, cpu int) {
 }
 
 func runMonitor() {
+	if os.Getuid() != 0 {
+		fmt.Fprintf(os.Stderr, "Please run the monitor with root privileges.\n")
+		os.Exit(1)
+	}
+
 	events, err := bpf.NewPerCpuEvents(&eventConfig)
 	if err != nil {
 		panic(err)

